source/class/qx/tool/compiler/ClassFile.js

```diff
@@ -2906,20 +2909,17 @@ qx.Class.define("qx.tool.compiler.ClassFile", {
       }
       return info;
     },
-
     /**
      * Adds a required asset
      * @param path
-     */
-    _requireAsset(path) {
+     */ _requireAsset(path) {
```

Comment:
I wonder why formatting is broken here or it is on purpose?

Reply:
ah - bug in prettier, i think you'll find that an earlier function had only a single line comment in it